components/collector/tests/source_collectors/dependency_track/test_security_warnings.py

```diff
@@ -46,20 +60,45 @@ async def test_no_projects(self):
 
     async def test_one_project_without_vulnerabilities(self):
         """Test one project without vulnerabilities."""
-        response = await self.collect(get_request_json_side_effect=[self.projects, []])
+        response = await self.collect(get_request_json_side_effect=[self.projects(), []])
         self.assert_measurement(response, value="0", entities=[])
 
     async def test_one_project_with_vulnerabilities(self):
         """Test one project with vulnerabilities."""
-        response = await self.collect(get_request_json_side_effect=[self.projects, self.vulnerabilities])
-        self.assert_measurement(response, value="1", entities=self.entities)
+        response = await self.collect(get_request_json_side_effect=[self.projects(), self.findings()])
+        self.assert_measurement(response, value="1", entities=self.entities())
 
-    async def test_filter_vulnerabilities(self):
+    async def test_filter_by_severity(self):
         """Test that vulnerabilities can be filtered."""
         self.set_source_parameter("severities", ["High", "Critical"])
-        response = await self.collect(get_request_json_side_effect=[self.projects, self.vulnerabilities])
+        response = await self.collect(get_request_json_side_effect=[self.projects(), self.findings()])
         self.assert_measurement(response, value="0", entities=[])
 
+    async def test_filter_by_project_name(self):
```

Comment:
If we make a DependencyTrackBaseTest that includes the filter tests, they'll be inherited by all these test classes

Reply:
I'm getting some weird unit test failures in unrelated parts of the code when introducing a base module for Dependency-Track unit tests, so I'm going to mark this as a TODO.